imports/plugins/included/paystack/client/checkout/paystack.js

```diff
@@ -0,0 +1,111 @@
+/* eslint camelcase: 0 */
+/* eslint no-undef: 0 */
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { Reaction } from "/client/api";
+import { Cart, Shops, Accounts, Packages } from "/lib/collections";
+
+import "./paystack.html";
+
+const getExchangeRate = () => {
+  const shop = Shops.find(Reaction.getShopId()).fetch();
+  return shop[0].currencies.NGN.rate;
+};
+
+const generateTransactionID = () => {
+  // generate a random 16 digit id for the transaction
+  return Random.id(16);
+};
+
+const getOrderPrice = () => {
+  const cart = Cart.findOne();
+  const exchangeRate = getExchangeRate();
+  return parseInt(cart.cartTotal() * exchangeRate, 10);
+};
+
+const getPayStackSettings = () => {
+  const packages = Packages.findOne({
+    name: "paystack-paymentmethod",
+    shopId: Reaction.getShopId()
+  });
+  return packages;
+};
+
+const finalizePayment = (payStackMethod) => {
+  Meteor.call("cart/submitPayment", payStackMethod);
+};
+
+handlePayment = (transactionId, type) => {
+  const payStackConfig = getPayStackSettings();
+  HTTP.call("GET", `https://api.paystack.co/transaction/verify/${transactionId}`, {
+    headers: {
+      Authorization: `Bearer ${payStackConfig.settings.secretKey}`
+    }
+  }, function (error, response) {
```

Comment:
Use arrow function here for consistency.

I would suggest you extract this function as a named function and simple pass in the function here. Something like `function paymentResponse(){}`